sw/vendor/riscv_compliance/riscv-target/opentitan/device/rv32imc/Makefile.include

```diff
@@ -0,0 +1,64 @@
+## Copyright lowRISC contributors.
+## Licensed under the Apache License, Version 2.0, see LICENSE for details.
+## SPDX-License-Identifier: Apache-2.0
+
+
+OT           = $(ROOTDIR)/riscv-target/$(RISCV_TARGET)/device/rv32imc
+OTSW         = $(ROOTDIR)/../../device
+OTROOT       = $(OTSW)/../../
+LDSCRIPT     = $(OT)/link.ld
+TRAPHANDLER  = $(OT)/handler.S
+DEFINES      = $(CARG) -DPRIV_MISA_S=0 -DPRIV_MISA_U=0 -DTRAPHANDLER="\"$(TRAPHANDLER)\""
+RV_TOOLS    ?= /tools/riscv/bin
+TARGET_SIM  ?= $(OTROOT)/build/lowrisc_systems_top_earlgrey_verilator_0.1/sim-verilator/Vtop_earlgrey_verilator
+TARGET      ?= fpga
+
+ifeq ($(TARGET),fpga)
+	CARG =
+	MAKEARG =
+	PYTEST_OPT = --fpga_uart /dev/ttyUSB2 --spiflash $(OTROOT)/sw/host/spiflash/spiflash \
+		--test_bin $(work_dir_isa)/$<.bin
+else
+	CARG = -DSIMULATION=1
+	MAKEARG = SIM=1
+	PYTEST_OPT = --verilator_model $(TARGET_SIM) --test_bin $(work_dir_isa)/$<.vmem \
+		--rom_bin $(OTSW)/boot_rom/rom.vmem
+endif
+
+
+RUN_TARGET=\
+        pytest -s -v $(OTROOT)/test/systemtest/functional_$(TARGET)_test.py \
+        $(PYTEST_OPT) \
+        --log $(work_dir_isa)/$<.uart.log; \
+        grep -o 'SIG: [a-zA-Z0-9_]*' $(work_dir_isa)/$<.uart.log | sed 's/SIG: //' \
+        > $(work_dir_isa)/$(*).signature.temp.output; \
+        tr '[:upper:]' '[:lower:]' < $(work_dir_isa)/$(*).signature.temp.output > $(work_dir_isa)/$(*).signature.output; \
```

Comment:
i'm not particularly skilled in make, but i did it this way mostly because of the how the riscv-compliance makefile is invoking this one. 

```
%.out32: %.elf
	$(RUN_TARGET)
```

let me dig around to see if there's a way to create another rule out of that sequence?